packages/env/lib/config/parse-config.js

```diff
@@ -85,6 +87,7 @@ const DEFAULT_ENVIRONMENT_CONFIG = {
 	themes: [],
 	port: 8888,
 	testsPort: 8889,
+	testsMysqlPort: null,
```

Comment:
I view the top-level `port` and `testsPort` properties are more of a legacy thing. It's more intuitive to set configuration like this _only_ at the environment level. How would you feel about dropping the root `testsMysqlPort` in favor of just having `development.mysqlPort` and `tests.mysqlPort`?